docs/upgrade.html

```diff
@@ -56,6 +56,8 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
                         <li>The <code>org.apache.kafka.common.security.oauthbearer.secured.OAuthBearerValidatorCallbackHandler</code> class was removed.
                             Please use the <code>org.apache.kafka.common.security.oauthbearer.OAuthBearerValidatorCallbackHandler</code> class instead.
                         </li>
+                        <li>The <code>org.apache.kafka.common.security.PasswordEncoderConfigs</code> class was removed.
```

Comment:
Could you please list the removed configs instead?

Reply:
I have modified the doc as following: 
![Screenshot from 2024-12-30 21-15-08](https://github.com/user-attachments/assets/cfb30710-c073-4732-819c-be293724f09b)